tests/unit/test_bootstrap.py

```diff
@@ -0,0 +1,118 @@
+import jubilant
+
+from . import mocks
+
+
+def test_defaults_interactive(run: mocks.Run):
```

Comment:
Per above, we should drop interactive support. So this would become just `test_defaults` and be something like `juju.bootstrap('microk8s')`.